WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -790,32 +785,20 @@ public void handleOnBackPressed() {
     }
 
     private void customizeToolbar() {
-        // Custom overflow icon
-        Drawable overflowIcon = ContextCompat.getDrawable(this, R.drawable.more_vertical);
```

Comment:
>Ah good point, I forgot to ask you about this icon, do we really need it to be that custom? It looks almost exactly like the default Android overflow menu icon so I ended up removing it.

Yeah we are trying to be consistent with the Gutenberg core icons within the editor, I know they are very similar 😄 

>If we need it then this would be the correct way of setting it, as it seems the overflowIcon can only be set by code.

Ok so it if'ts ok with you I'll restore that part of the code to keep the custom overflowIcon 👍